### DISCUSSION TITLE
how the placement of UI elements works?

### DISCUSSION THREAD

**Discussion by j-a-v-i-c-k** [FakeFishGames/Barotrauma#12803] [Code questions and modding support]
Hello, could someone please describe how the placement of UI elements works.

For example:
GuiFrame relative="0.15,0.35" anchor="Center" pivot="Center" relativeoffset="0.84,0.65" style="ItemUI"

What is “pivot” for and how to configure “relativeoffset” correctly?
Thanks in advance.

**Comment by iridescent-lightning** [comment]
Relative offset means to adjust offset based on parent gui width and length
